pkg/apis/k8s/v1alpha1/nginxingresscontroller_types.go

```diff
@@ -40,13 +40,17 @@ type NginxIngressControllerSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	EnableSnippets bool `json:"enableSnippets"`
-	// +kubebuilder:validation:Optional
 	// A class of the Ingress controller. The Ingress controller only processes Ingress resources that belong to its
 	// class (in other words, have the annotation “kubernetes.io/ingress.class”).
 	// Additionally, the Ingress controller processes Ingress resources that do not have that annotation,
 	// which can be disabled by setting UseIngressClassOnly to true. Default is `nginx`.
+	// +kubebuilder:validation:Optional
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	IngressClass string `json:"ingressClass"`
+	// Specifies extra labels of the service.
```

Comment:
could you also update this doc https://github.com/nginxinc/nginx-ingress-operator/blob/master/docs/nginx-ingress-controller.md to include new fields ?